### PR TITLE
Infection icon now has lower alpha if the human is dead

### PR DESCRIPTION
**Pull request by ihatethisengine**
# About the pull request
So this was bothering me for a long time and now I finally found a simple solution.

I cannot make 5 new sprites to display that hugged person is dead.

I cannot change the color of the sprite, as you need to know which hive larva belongs to.

But I can lower alpha! Simple solution.

# Explain why it's good for the game
Makes it less confusing for both xenos and observers to figure if the hugged person is dead. Sometimes xenos ignore people who stepped in a resin trap because they assume that the person is dead, why would the person be lying in the middle of nowhere otherwise?

# Testing Photographs and Procedure

# Changelog

:cl: ihatethisengine
add: Infection icon now has lower alpha if the human is dead
/:cl:

### CODE REVIEW

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Comment by ihatethisengine** [comment]
Unreal conflict with my own PR

**Comment by ihatethisengine** [comment]
Thanks for helping me with the conflict, web-editor :roll_eyes:

**Comment by HeresKozmos** [comment]
Pretty neat.

**Review by realforest2001 — APPROVED** [review]

**Review by Drulikar — CHANGES_REQUESTED** [review]
This needs to also reset the alpha if they then are revived. Simple solution would be to else set the alpha to full.

File: code/datums/mob_hud.dm [hunk 1]
```diff
@@ -422,6 +422,10 @@ GLOBAL_LIST_INIT_TYPED(huds, /datum/mob_hud, list(
 				if(hive && hive.color)
 					holder3.color = hive.color
 
+				if(stat == DEAD || status_flags & FAKEDEATH)
+					holder2.alpha = 100
+					holder3.alpha = 100
+
```
Comment (ihatethisengine) [inline code/datums/mob_hud.dm:428]:
```suggestion
				if(stat == DEAD || status_flags & FAKEDEATH)
					holder2.alpha = 100
					holder3.alpha = 100
				else
					holder2.alpha = 255
					holder3.alpha = 255
```

**Review by Drulikar — APPROVED** [review]
I will note is also leaves the death icon transparent if they are a xeno host, but this is probably fine.
image

**Pull request merged**